.golangci.yml

```diff
@@ -39,8 +39,6 @@ issues:
     # disable typecheck in folder where it breaks because of build tags
     - path: "pkg/security/"
       linters: [typecheck]
-    - path: "pkg/process/"
```

Comment:
created a [task](https://datadoghq.atlassian.net/browse/ADXT-494) for global reassessment of linter exclusions